src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/alerts/AlertMapping.kt

```diff
@@ -0,0 +1,54 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.alerts
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.data.annotation.Id
+import org.springframework.data.annotation.Transient
+import org.springframework.data.domain.Persistable
+import java.time.LocalDateTime
+
+data class AlertMapping(
+
+  @Id
+  val dpsAlertId: String,
+
+  val nomisBookingId: Long,
+  val nomisAlertSequence: Long,
+
+  /**
+   * ISO timestamp of batch job if a migration
+   */
+  val label: String? = null,
+
+  val mappingType: AlertMappingType,
+
+  @Transient
+  @Value("false")
+  val new: Boolean = true,
```

Comment:
Why are these opposites?

Reply:
I have no idea @mikehalmamoj  - all the entities are like this. I have a feeling there is a good reason, I just don't know what it is

Reply:
I will investigate